prac_06/guitars.py

```diff
@@ -0,0 +1,15 @@
+from prac_06.guitar import Guitar
+
+guitars = []
+print("My guitars!")
+name = input("Name: ")
+while name != "":
+    year = int(input("Year: "))
+    cost = float(input("Cost: $"))
+    guitars.append(Guitar(name, year, cost))
+    print("")
+    name = input("Name: ")
+print("These are my guitars: ")
+for i, guitar in enumerate(guitars):
```

Comment:
You can set the starting value for `i` here as 1, using:
`for i, guitar in enumerate(guitars, 1):`
which saves you having to add 1 to the `i` value in the print line below

Reply:
Thanks, fixed.